api/db-cdr/changelog-data/db.changelog-1.xml

```diff
@@ -29,18 +29,6 @@
     <delete tableName="ds_data_dictionary"/>
     <delete tableName="domain_card"/>
```

Comment:
Need to remove the <delete tableName="domain_info"/> line above or load local data command will fail after the first run

Reply:
Ok, done.